### DISCUSSION TITLE
Companies using CodeSandbox

### DISCUSSION THREAD

**Discussion by CompuIves** [codesandbox/codesandbox-client#236] [General] [answered]
Does your company use CodeSandbox? Respond here if you do!

Something along the lines of:

Company name:
What you use it for:
Would you want to be featured on the homepage:

**Comment by Xananax** [comment]
We teach full stack web dev in a coding bootcamp called [Codi](http://codi.tech). We use CodeSandbox daily to prototype things, show examples, start little projects that the students can then copy and improve on, and maybe more importantly allow students to develop applications without having to worry about setting up a webpack toolchain.

We don't yet use it this way, but I think it's also a great tool to introduce different frameworks and/or languages (Typescript & al) with minimal friction.

I wouldn't mind Codi being featured on the homepage (but no pressure either to do so)

**Comment by grant** [comment]
Company name: Google
What you use it for: [Apps Script](https://developers.google.com/apps-script/) development with [clasp](https://github.com/google/clasp/).
Would you want to be featured on the homepage: Eventually, if the use case with Google worked. Probably a Google Cloud feature. Would need legal approval on my side.

It looks like some further additions would be required to make a good devX in codesandbox:
- Support `tsconfig.json` for better autocomplete results. https://github.com/CompuIves/codesandbox-client/issues/1207
- Support fully hiding web view panel.
- Adding a new deploy method for Google APIs. [Looks pretty straightforward](https://github.com/CompuIves/codesandbox-client/search?q=zeit&unscoped_q=zeit)
- New configuration file (`.clasp.json`)
- Probably some other things, but those are the basics.

**Comment by Saeris** [comment, marked as answer]
We currently have a list of companies using CodeSandbox on the homepage with a link to the file in which users can add themselves:

https://codesandbox.io/who-uses-codesandbox

https://github.com/codesandbox/codesandbox-client/blob/master/packages/homepage/content/users.json

Closing this for now.
